packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -264,6 +273,11 @@ class _CodeViewState extends State<CodeView>
             child: Expanded(
               child: Scrollbar(
                 controller: textController,
+                // Only listen for vertical scroll notifications (ignore those
```

Comment:
why are we getting notifications for the horizontal scoll controller? It looks like we are specifying just the textController which is the vertical scroll controller although it could use a better name.

Reply:
From the Scrollbar class [docs](https://api.flutter.dev/flutter/material/Scrollbar-class.html): "If the scrollbar is wrapped around multiple ScrollViews, it only responds to the nearest scrollView and shows the corresponding scrollbar thumb by default. The notificationPredicate allows the ability to customize which ScrollNotifications the Scrollbar should listen to."

The horizontal scrollview is the first nested scrollview, and then the vertical scrollview (see image). So this is to ignore notifications from the horizontal scrollview. Other option would be to try to change the nesting, so the list view is the first scrollable child under the vertical scrollbar.

![image](https://user-images.githubusercontent.com/21270878/129096776-c8afcf6c-08c3-47fe-aabc-28b840b37121.png)

Reply:
I thought that is only the case if you do not specify a scrollController explicitly like you are.
Perhaps the Flutter docs should be updated to be more clear on that or I am misunderstanding something.

Reply:
Resolving this in lieu of the other comment thread about the `notificationPredicate`